include/seqan3/alphabet/container/concatenated_sequences.hpp

```diff
@@ -1327,7 +1228,10 @@ class concatenated_sequences
      * \tparam rng_type The type of range to be inserted; must satisfy \ref is_compatible_with_value_type.
      * \param value The value to append.
      *
-     * If the new size() is greater than capacity() then all iterators and references (including the past-the-end
+     * This conceptionally adds another element to the container with the specified content,
+     * i.e. the "outer container" grows by 1.
+     *
+     * If the new concat_size() is greater than concat_capacity() then all iterators and references (including the past-the-end
      * iterator) are invalidated. Otherwise only the past-the-end iterator is invalidated.
```

Comment:
```suggestion
     * If the new concat_size() is greater than concat_capacity(), all iterators and references (including the 
     * past-the-end iterator) are invalidated. Otherwise, only the past-the-end iterator is invalidated.
```